packages/engine/Source/Core/Color.js

```diff
@@ -352,9 +352,9 @@ const rgbaMatcher = /^#([0-9a-f])([0-9a-f])([0-9a-f])([0-9a-f])?$/i;
 //#rrggbbaa
 const rrggbbaaMatcher = /^#([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})?$/i;
 //rgb(), rgba(), or rgb%()
-const rgbParenthesesMatcher = /^rgba?\(\s*([0-9.]+%?)\s*,\s*([0-9.]+%?)\s*,\s*([0-9.]+%?)(?:\s*,\s*([0-9.]+))?\s*\)$/i;
+const rgbParenthesesMatcher = /^rgba?\(\s*([0-9.]+%?)\s*[,\s]\s*([0-9.]+%?)\s*[,\s]\s*([0-9.]+%?)(?:\s*[,\s/]\s*([0-9.]+))?\s*\)$/i;
```

Comment:
```suggestion
const rgbParenthesesMatcher = /^rgba?\(\s*([0-9.]+%?)\s*[,\s]\s*([0-9.]+%?)\s*[,\s]\s*([0-9.]+%?)(?:\s*[,\s]\s*([0-9.]+))?\s*\)$/i;
```

?

Reply:
No, it needs for correct alpha parsing

Eg. `rgb(255 255 255 / .5)`

Reply:
Ah, then the character need to be escaped- `\/`

Reply:
I checked this regexp in browser console, also in Node.js 16, it works. Seems it doesn't in Node.js 18...

Reply:
@ggetz, I tried escaping - didn't help.
That I removed `/` from regexp and corresponding tests - 3rd stage now passed.

I tested all my regexps in browser - it works, but tests doesn't.
Could you please check these tests locally? I have some problems with local test running

Reply:
Thank you!